packages/portal-proto/src/features/set-operations/CountButtonWrapperForSetsAndCases.tsx

```diff
@@ -135,7 +135,6 @@ const CountButtonWrapperForSetsAndCases: React.FC<
 
   const createCohort = async () => {
     return await createSet({
-      // TODO: possibly add error handling
```

Comment:
Is this done?

Reply:
We should. However, we need to be consistent about how we handle such errors. I propose we do this in a separate ticket.

Reply:
yeah, definitely a separate ticket :)